asciidoc-confluence-publisher-doc/etc/docs/00-index/05-tables.adoc

```diff
@@ -101,3 +101,61 @@ Currently, only simple formattings, but no nested tables or lists are supported.
 | *B1*
 | *B2*
 |===
+
+== Table with Row Span
+
+[listing]
+....
+[cols="2", options="header"]
+|===
+| Column 1
+| Column 2
+
+.2+| A1
+| A2
+| A3
+
+| B1
+| B2
+|===
+....
+
+[cols="2", options="header"]
+|===
+| Column 1
+| Column 2
+
+.2+| A1
+| A2
+| A3
+
+| B1
+| B2
+|===
+
+== Table with Col Span
+
+[listing]
+....
+[cols="2", options="header"]
```

Comment:
Using an example with three columns would make it clear that col spans do not have to span all columns of a table (similar to row span example).